code/game/gamemodes/cult/narsie.dm

```diff
@@ -95,12 +95,15 @@ GLOBAL_LIST_EMPTY(narsie_list)
 	else if(istype(A, /obj/structure/cult))
 		qdel(A)
 
-/obj/singularity/narsie/large/CollidedWith(atom/A)
-	if(!cause_hell) return
-	if(isturf(A))
-		narsiewall(A)
-	else if(istype(A, /obj/structure/cult))
-		qdel(A)
+/obj/singularity/narsie/large/CollidedWith(atom/movable/bumped_atom)
+	. = ..()
+
+	if(!cause_hell)
+		return
+	if(isturf(bumped_atom))
```

Comment:
bumped_atom is a atom/movable
But turfs are not.

https://www.byond.com/docs/ref/#/atom

Reply:
Looks like it's old Bay thing, apparently they have some code from 14 years ago where a turf could collide with another (???)

Either way we are thankfully slightly more sane so this doesn't look like to be possible, I'll smite it